test/openlr.cc

```diff
@@ -153,14 +141,14 @@ TEST(OpenLR, InternalReferencePoints) {
 
 TEST(OpenLR, OffsetsOverrun) {
   auto location = "CwG1ASK3PhD82sz0CIAQ89oAAAAAEPPa/N4UcRBj";
-  auto locRef = LineLocation(decode64(location));
+  auto locRef = LineLocation(location, true);
   EXPECT_EQ(locRef.poff, 0.f);
   EXPECT_EQ(locRef.noff, 0.f);
 }
 
 TEST(OpenLR, TooSmallReference) {
-  auto location = "CwG1ASK3PhD82=";
-  EXPECT_THROW(auto locRef = LineLocation(decode64(location)), std::invalid_argument);
+  auto location = "83hRxEAM=";
```

Comment:
can you comment about the change here?

Reply:
Added a comment to the code. The original reference here wasn't valid base64.